_delphi_utils_python/tests/test_export.py

```diff
@@ -372,3 +381,40 @@ def test_export_df_with_contradictory_missingness(self, mock_logger):
         mock_logger.info.assert_called_once_with(
             "Filtering contradictory missing code in test_None_2020-02-15."
         )
+
+    def test_export_sort(self):
+        _clean_directory(self.TEST_DIR)
+
+        unsorted_df = pd.DataFrame({
+            "geo_id": ["51175", "51093", "51175", "51620"],
+            "timestamp": [
+                datetime.strptime(x, "%Y-%m-%d")
+                for x in ["2020-02-15", "2020-02-15", "2020-03-01", "2020-03-15"]
+            ],
+            "val": [3.12345678910, 2.1, 2.2, 2.6],
+            "se": [0.15, 0.22, 0.20, 0.34],
+            "sample_size": [100, 100, 101, 100],
+        })
+        create_export_csv(
+            unsorted_df,
+            export_dir=self.TEST_DIR,
+            geo_res="county",
+            sensor="test"
+        )
+        assert_frame_equal(
+            _set_df_dtypes(pd.read_csv(join(self.TEST_DIR, "20200215_county_test.csv")), dtypes={"geo_id": str}),
+            unsorted_df.query("timestamp == '2020-02-15'").drop(columns="timestamp").reset_index(drop=True)
```

Comment:
The expected df only has two rows, so it may actually be clearer here to write it out instead of computing it